post.md

```diff
@@ -57,7 +57,7 @@ It's easy to use:
 ~~~
```

Comment:
It sort of sounds like just a basic loading spinner wrapper at the high level - can you explain why the rendering here is special at a high level (or is that pre-requisite knowledge) before we dig into the details? 

`<Component>
 isDataLoaded ? <Inner gubbins/> : <Loading spinner/>
</Component>`